### PR TITLE
Color sink

### PR DESCRIPTION
**Pull request by NikhilAnand1998**
Added the functionality that changes an opponents ship to red when they sink your ship.

### CODE REVIEW

File: src/main/java/assets/css/custom.css [hunk 1]
```diff
@@ -177,6 +177,8 @@ td:hover {
     background-color: blue;
 }
 
+
```
Comment (aalokb) [inline src/main/java/assets/css/custom.css]: Fix the spacing here, its a wee bit inconsistent

File: src/main/java/assets/game.js [hunk 2]
```diff
@@ -60,19 +65,105 @@ function makeAxis(table, is_x) {
...
+                if(attack.result === "SURRENDER"){
+                          $("#hit_numb").text("Accuracy: " + accuracy + "%");
+                          $("#sink_numb").text("Ships Sunk: " + ship_sunk );
+                           $("#end_game_modal").show();
+                }
+
+
+
         }
-        else if (attack.result === "SUNK"){
```
Comment (aalokb) [inline src/main/java/assets/game.js]: I think by removing this we reduced the redundancy of the ship sinking/surrendering, this is solid. Now we just have to remember that sink and surrender enter the same conditional statement, upon attack...

Comment (aalokb) [reply src/main/java/assets/game.js]: @NikhilAnand1 @NikhilAnand1998

Comment (kramerje) [reply src/main/java/assets/game.js]: I like this as well, nice.

**Comment by thenellg** [comment]
Everything looks good on my end. Frankly, not seeing anything that wasn't already talked about.

**Pull request merged**